packages/devops/scripts/lambda/actions/redeploy_tupaia_server.py

```diff
@@ -85,7 +85,6 @@ def redeploy_tupaia_server(event):
             deployment_name=get_tag(existing_instance, 'DeploymentName'),
             branch=get_tag(existing_instance, 'Branch'),
             instance_type=event.get('InstanceType', existing_instance['InstanceType']),
-            extra_tags=extra_tags + [{ 'Key': 'DeploymentComponent', 'Value': 'app-server' }], # TODO remove deployment component stuff after move to RDS
```

Comment:
We do need the extra tags in this and the next file for things other than the DeploymentComponent